### ISSUE TITLE
Antd styles' component not being applied

### ISSUE THREAD

**Issue by richardaum** [joelaguero/react-parcel-ssr-antd#3]
Unfortunately, this repo isn't applying styles from `antd`, probably to some misconfiguration related to CSS modules.

![image](https://user-images.githubusercontent.com/2018161/82510187-60773b80-9ae0-11ea-8088-5339f5d46b79.png)

Test button should be have antd styles.

**Comment by joelaguero** [comment]
Yeah this is probably outdated now unfortunately. Apologies for the inconvenience. If I have time I’ll try to put together a new boilerplate for you!
